src/index.js

```diff
@@ -0,0 +1,19 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <Router>
+    <App />
+  </Router>,
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
```

Comment:
Kindly remove unnecessary comments. Removing these kinds of comments will make your code look more professional and also is going to save you some lines of code.

Reply:
@sebalp100 Thank you for reviewing my code and leaving me notes. I have removed all the comments.